nodejs/nodejs-cloud-run-hello-world/package-lock.json

```diff
@@ -1201,7 +1201,7 @@
         "extend": "^3.0.2",
         "https-proxy-agent": "^5.0.0",
         "is-stream": "^2.0.0",
-        "node-fetch": "^2.3.0"
+        "node-fetch": "^2.6.1"
```

Comment:
Should the package-lock be generated via `npm install`? Is it okay we update it manually?